client/src/views/Simulation/Simulation.vue

```diff
@@ -268,6 +293,30 @@
     onNodeHighlight (label: string): void {
       this.highlighted = label;
     }
+
+    onShowProvenanceGraph (): void {
```

Comment:
Thanks for the suggestion, maybe a better name is `onToggleProvenanceGraph`? Since the Provenance Graph button acts as a toggle, we need to keep track of the current state of the graph. So when it is already open (true), the graph should close when the user clicks on the button. When it is closed (false), the graph should open when the user clicks on the button.